vrecord

```diff
@@ -532,7 +541,12 @@ _setup_vrecord_input(){
 _setup_vrecord_process(){
     _setup_vrecord_input
     if [[ "${DEVICE_INPUT_CHOICE}" = 0 ]] ; then
-        PIPE_OUTPUT=(-c:v copy)
+        if [[ "${SIGNAL_INPUT}" != 'true' ]] ; then
+            PIPE_OUTPUT=(-c:v copy)
+        fi
+        if [[ "${SIGNAL_INPUT}" = 'true' ]] ; then
+            PIPE_OUTPUT+=(-map 1)
+        fi
```

Comment:
Does this 
```Bash
        if [[ "${SIGNAL_INPUT}" != 'true' ]] ; then
            PIPE_OUTPUT=(-c:v copy)
        else
            PIPE_OUTPUT+=(-map 1)
        fi
```
not work?

Reply:
ah - yeah good catch! I forgot to squish those back together after pulling them apart to do some more granular testing on adding options!